crates/node/service/src/service/standard/builder.rs

```diff
@@ -160,6 +167,7 @@ impl RollupNodeBuilder {
             p2p_config: self.p2p_config,
             network_disabled: self.network_disabled,
             runtime_launcher,
+            ws_enabled: self.ws_enabled,
```

Comment:
We can omit this from the `RollupNode` and set it on the `rpc_launcher` we build a little earlier in this function.

Take a look at like 143:
```rust
        let rpc_launcher = self.rpc_config.map(|c| c.as_launcher()).unwrap_or_default();
```

Since you added the `with_ws_enabled` method on the `RpcLauncher`, we can just set this here in the builder like so.

```rust
        let rpc_launcher = self.rpc_config.map(|c| c.as_launcher()).unwrap_or_default().with_ws_enabled(self.ws_enabled);

```